src/App/Tracker/Controller/Upload/Ajax/Put.php

```diff
@@ -42,20 +42,33 @@ protected function prepareResponse()
 			$destDir    = $application->getProject()->project_id;
 			$fullPath   = $host . '/' . $application->get('system.upload_dir') . '/' . $destDir . '/' . $destName;
 
+			$isImage = (false === strpos($file->getMimetype(), 'image')) ? false : true;
```

Comment:
Made it as `$isImage = strpos($file->getMimetype(), 'image');`
Should work also ;)